index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>A Solutions</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header>
+        <div class="container flex-box ">
+            <img src="images/logo.png" alt="A Solutions" id="logoImage">
+            <div class="nav-bar flex-box">
+                <div class="icon-bar flex-box">
+                    <i class="fa fa-search" id="icon"></i>
+                    <i class="fa fa-user" id="icon"></i>
+                    <i class="fa fa-envelope" id="icon"></i>
+                </div>
+                <nav>
+                   <ul class="flex-box">
+                      <li><a href="#home">Home</a></li>
+                      <li><a href="#about">About</a></li>
+                      <li><a href="#services">Services</a></li>
+                      <li><a href="#training">Training</a></li>
+                      <li><a href="#contact">Contact</a></li>
+                    </ul> 
+                </nav>
+            </div>    
+        </div>
+    </header>
+    <main>
+        <section class="banner flex-box">
+            <div class="content flex-box">
+                <div class="panel-left">
+                    <img src="images/about.png" alt="About">
+                </div>
+                <div class="panel-right">
+                    <h2>About A Solutions</h2>
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    Lorem Ipsum has been the industry's standard dummy text ever since the 
+                    1500s, when an unknown printer took a gallery of type and scrambled it to make.</p> 
+                </div>
+            </div>
+        </section>
+        <div class="main-content">
```

Comment:
use relevant class name .  Also , try to maintain uniformity in the HTML tags . About is using a 'section' but the next one is just a div

Reply:
Use proper  background  color to distinguish two sections , there should be no white gap in between